src/Http/OpenApi/src/Microsoft.AspNetCore.OpenApi.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageTags>aspnetcore;openapi</PackageTags>
+    <Nullable>enable</Nullable>
+    <Trimmable>true</Trimmable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.OpenApi" />
+    <Reference Include="Microsoft.AspNetCore.Http.Abstractions" />
+    <Reference Include="Microsoft.AspNetCore.Routing" />
+    <Reference Include="Microsoft.AspNetCore.Mvc.Core" />
```

Comment:
Organization question: Why put this under /Http/ when it depends on higher level components like MVC?